<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenLIFUData</class>
 <widget class="qMRMLWidget" name="OpenLIFUData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>446</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="sessionsCollapsibleButton">
     <property name="text">
      <string>Sessions</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="databaseDirectoryLabel">
        <property name="text">
         <string>Database directory:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ctkPathLineEdit" name="databaseDirectoryLineEdit">
        <property name="filters">
         <set>ctkPathLineEdit::Dirs</set>
        </property>
        <property name="SlicerParameterName" stdset="0">
         <string>databaseDirectory</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="databaseLoadButton">
        <property name="text">
         <string>Load Database</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="subjectSessionLabel">
        <property name="text">
         <string>Subject/session selector:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTreeView" name="subjectSessionView">
        <property name="expandsOnDoubleClick">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sessionLoadButton">
        <property name="text">
         <string>Load</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkPathLineEdit</class>
   <extends>QWidget</extends>
   <header>ctkPathLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
